src/components/steps/step-sheet/DetailsTab.tsx

```diff
@@ -21,23 +23,46 @@ export function StepDetailsTab({ stepId, runId }: Props) {
 	if (isError) return <ErrorFallback err={error} />;
 	if (isPending) return <Skeleton className="h-[300px] w-full" />;
 
+	const transformToEllipsis = (text: string, maxLength: number) => {
```

Comment:
I think this should be a function from the `lib` directory. It's not specifically tied to this component I think